### PR TITLE
fix: Fix bundling

### PR DESCRIPTION
**Pull request by imprashast**
This'll fix the bundling issues we had.

Here's some context:

> Right now the package is bloated & the Translations doesn’t work as we don’t have the src folder in the package. Meanwhile we are also building and bundling the react/vue/elements files using esbuild. Those are the only files that should be distributed + the raw icons.

### CODE REVIEW

File: vue.eik.js [hunk 1]
```diff
@@ -1,10 +1,6 @@
-import * as eik from '@eik/esbuild-plugin';
 import esbuild from 'esbuild';
 
-await eik.load();
-
 await esbuild.build({
-  plugins: [eik.plugin()],
```
Comment (BalbinaK) [inline vue.eik.js:7]: Why was this removed? Now in the output icons.js we have `import sth from 'react'` instead of an eik url 🤔 Was that intended?

Comment (imprashast) [reply vue.eik.js:7]: Since this is an npm package being used in project that are either `react` or `vue`, they are already importing react or vue, and probably bundling their code. Eik in that project would take care of that. Doing this here was breaking icons in multiple apps because esbuild doesn't like packages importing from `http:` :)
